backtracepython/report.py

```diff
@@ -11,33 +11,31 @@
 
 
 def add_source_code(source_path, source_code_dict, source_path_dict, line):
-    try:
-        the_id = source_path_dict[source_path]
-    except KeyError:
-        the_id = str(uuid.uuid4())
-        source_path_dict[source_path] = the_id
-        source_code_dict[the_id] = {
+
+    if source_path in source_path_dict:
+        source_code_info = source_code_dict[source_path]
+        if line < source_code_info["minLine"]:
+            source_code_info["minLine"] = line
+        if line > source_code_info["maxLine"]:
+            source_code_info["maxLine"] = line
+
+    else:
+        source_code_dict[source_path] = {
             "minLine": line,
             "maxLine": line,
             "path": source_path,
         }
-        return the_id
 
-    if line < source_code_dict[the_id]["minLine"]:
-        source_code_dict[the_id]["minLine"] = line
-    if line > source_code_dict[the_id]["maxLine"]:
-        source_code_dict[the_id]["maxLine"] = line
-    return the_id
+    return source_path
 
 
 def process_frame(tb_frame, line, source_code_dict, source_path_dict):
     source_file = os.path.abspath(tb_frame.f_code.co_filename)
     frame = {
         "funcName": tb_frame.f_code.co_name,
         "line": line,
-        "sourceCode": add_source_code(
-            source_file, source_code_dict, source_path_dict, line
-        ),
+        "library": source_file,
```

Comment:
maybe it would be better to use filename here only?

Reply:
The source code integration (with the source code service and source code injection in the client side) won't work